File: 프로그래머스/lv1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.java
class Solution {
    public String solution(String[] seoul) {
        String result = null;
        for (int i = 0; i < seoul.length; i++) {
            if (seoul[i].equals("Kim")) {
                result = String.valueOf(i);
                break;
            }
        }
        return  "김서방은 " + result + "에 있다";
    }
}

File: 프로그래머스/1/12919. 서울에서 김서방 찾기/서울에서 김서방 찾기.kt
class Solution {
    fun solution(seoul: Array<String>): String {
        var answer = ""
        seoul.forEachIndexed { i, it ->
            if (it == "Kim") {
                answer = "김서방은 ${i}에 있다"
            }
        }
        return answer
    }
}
